remote-api/src/main/java/org/alfresco/repo/webdav/WebDAVLockServiceImpl.java

```diff
@@ -444,35 +462,27 @@ private static final boolean removeObjectFromSessionList(HttpSession session, St
      * @param nodeRef NodeRef
      * @param userName String
      * @param createExclusive boolean
-     * @param timeoutSecs int
      */
-    private LockInfo createLock(NodeRef nodeRef, String userName, boolean createExclusive, int timeoutSecs)

```

Comment:
setting timeout seconds has been extracted to be passed directly instead of through setTimeoutSeconds()